README.md

```diff
@@ -1,4 +1,43 @@
-# Python boilerplate for GitHub tasks
+# User Permissions API
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+- Download [ModHeader](https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj?hl=en)
+
+### In this task you will add the functionality of user permissions
+
+1. Create serializers and views to support the following endpoints:
+   * `POST api/register/` - You can create here a user
+   * `POST api/login/` - You can get a token, if you write the correct data
+   * `POST api/me/` - Information about user and possibility to update information about user
+
+
+Example:
+```python
+HTTP 200 OK
+Allow: GET, PUT, PATCH, HEAD, OPTIONS
+Content-Type: application/json
+Vary: Accept
+
+{
+    "id": 1,
+    "username": "admin1",
+    "email": "",
+    "is_staff": true
+}
+```
+
+2. Create `cinema/permissions.py` with next API access:
```

Comment:
By default, all endpoints in API must have the following action limitations depending on the user role:
```suggestion
2. By default, all endpoints in API must have the following action limitations depending on the user role:
```

Reply:
done